.claude/skills/tool-currency-check/SKILL.md

```
@@ -0,0 +1,101 @@
+   ```bash
+   # step 0: local cache
+   grep -rHi <feature> docs/research/mintlify-cache/jdx/<tool>/
+   # step 1/2: remote, only on cache miss
+   curl -s https://raw.githubusercontent.com/jdx/<tool>/main/docs/... .md
+   ```
+
+   Look specifically for features that would let us **delete** custom code.
+
+5. **Emit a retire/bump report.** One table:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)